addons/mail/static/src/worklets/audio_processor.js

```diff
@@ -1,3 +1,41 @@
+class BiquadBandpassFilter {
+    constructor(sampleRate, frequency, Q) {
+        // TODO maybe comment each steps to match the standard biquad filter equation
+        this.x1 = 0;
+        this.x2 = 0;
+        this.y1 = 0;
+        this.y2 = 0;
+
+        const w0 = 2 * Math.PI * (frequency / sampleRate);
+        const cosW0 = Math.cos(w0);
+        const sinW0 = Math.sin(w0);
+        const alpha = sinW0 / (2 * Q);
+        const a0 = 1 + alpha;
+
+        this.b0 = alpha / a0;
+        this.b1 = 0;
+        this.b2 = -alpha / a0;
```

Comment:
See https://www.w3.org/TR/audio-eq-cookbook/#formulae for more information on the coefficients

Implemented the constant peak gain (b0=α) variant as it seems to be the better for voice activation use case.

alternative with constant skirt gain variant (not sure if better for this use case) would be:

```suggestion
        this.b0 = (alpha * Q) / a0;
        this.b1 = 0;
        this.b2 = -(alpha * Q) / a0;
```
or
```suggestion
        this.b0 = sinW0 / (2 * a0);
        this.b1 = 0;
        this.b2 = -sinW0 / (2 * a0)
```